requirements/prod-requirements.txt

```diff
@@ -3,7 +3,6 @@ tornado<5.0
 newrelic
 errand-boy==0.3.8
 setproctitle==1.1.9
-uWSGI==2.0.11.2
```

Comment:
non-blocking: is this a necessary part of this PR or just misc cleanup?

Reply:
That was unintentional. Thanks for catching it